modules/programs/just.nix

```
@@ -0,0 +1,64 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.programs.just;
+
+in {
+  meta.maintainers = [ maintainers.maximsmol ];
```

```suggestion
  meta.maintainers = [ hm.maintainers.maximsmol ];
```

Home Manager's maintainers are available `lib.hm.maintainers`.